sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/IncrementalExecution.scala

```diff
@@ -47,11 +47,16 @@ class IncrementalExecution(
       sparkSession.sparkContext,
       sparkSession.sessionState.conf,
       sparkSession.sessionState.experimentalMethods) {
-    override def extraPlanningStrategies: Seq[Strategy] =
-      StatefulAggregationStrategy ::
-      FlatMapGroupsWithStateStrategy ::
-      StreamingRelationStrategy ::
-      StreamingDeduplicationStrategy :: Nil
+    override def extraPlanningStrategies: Seq[Strategy] = {
+      //  We shouldn't miss the parent's strategies
+      val parentPlanner = sparkSession.sessionState.planner
+
+      parentPlanner.strategies ++ parentPlanner.extraPlanningStrategies ++
```

Comment:
Hive overrides the strategies method of the SparkPlanner. Maybe I should actually override the `strategies` method instead?